modules/ROOT/pages/_partials/endpoint-http-receive.adoc

```diff
@@ -0,0 +1,73 @@
+== HTTP Receive Settings
+
+These settings are a subset of those in xref:3.8@mule-runtime::http-listener-connector.adoc[HTTP Listener Connector].
+
```

Comment:
Why are we pointing to 3.8 doc here?

Reply:
We are not, it is an older version of the doc. Will update when it's in the product. I probably should have said that it was not ready to review. My mistake.